api/v1/models/MailingList.js

```diff
@@ -0,0 +1,27 @@
+var Model = require('./Model');
+var User = require('./User');
+var MailingListUser = require('./MailingListUser');
+
+var MailingList = Model.extend({
+	tableName: 'mailing_lists',
+	idAttribute: 'id'
+});
+
```

Comment:
addToList adds a user to a list right? If so we should make sure we are not adding duplicate users and that can go in addToList. But there will be more than one list (I would think) and so something like a unique constraint on the name of the list would be important as well.